grakn-graql/src/main/java/ai/grakn/graql/internal/gremlin/fragment/Fragment.java

```diff
@@ -99,4 +99,8 @@ default boolean isStartingFragment() {
     }
 
     double fragmentCost(double previousCost);
+
+    default boolean hasFixedFragmentCost() {
+        return false;
+    }
```

Comment:
I know I'm being a hypocrite, since the method above doesn't have it - but we should put some Javadoc for this method :P